### PR TITLE
Added batch queue example

### PR DESCRIPTION
**Pull request by Sharpz7**
Closes #https://github.com/G-Research/armada/issues/1420

Adds example code for:
- Creating or Updating a batch of queues.
- Cancelling a Job-Set with a filter

### CODE REVIEW

**Review by kannon92 — APPROVED** [review]

File: client/python/examples/queues.py [hunk 1]
```diff
@@ -9,11 +9,9 @@
 from armada_client.permissions import Permissions, Subject
 
 
-def creating_full_queues_example(client, queue):
+def create_queue_request(client, queue):
```
Comment (jayofdoom) [inline client/python/examples/queues.py:12]: any reason we aren't putting types in here, like we do for the code? (throughout)

**Review by jayofdoom — DISMISSED** [review]

File: client/python/examples/queues.py (hunk 1 above)
Comment (Sharpz7) [reply client/python/examples/queues.py:12]: Python is a dynamic language. I just feel that we do not want to enforce the typing standards we set for Python Armada Client on the people that use it.

File: client/python/examples/cancelling.py [hunk 2]
```diff
@@ -103,9 +126,9 @@ def quick_create_queue(client, queue):
 def workflow():
     """
     Starts a workflow, which includes:
-        - Creating a queue
-        - Creating a jobset
-        - Creating a job
+        - Creating a queue a job
```
Comment (jayofdoom) [inline client/python/examples/cancelling.py:129]: missing an and? Or something, this doesn't make sense as written

File: client/python/examples/queues.py (hunk 1 above)
Comment (jayofdoom) [reply client/python/examples/queues.py:12]: I can see that, but type hints don't break non-type hinted code. Either way, I'll trust your judgement here.

**Review by jayofdoom — COMMENTED** [review]
Please fix the one place with the nonsensical comment

**Review by kannon92 — APPROVED** [review]

File: client/python/examples/cancelling.py (hunk 2 above)
Comment (Sharpz7) [reply client/python/examples/cancelling.py:129]: Yeah this needs fixed. On it shortly

**Review by kannon92 — APPROVED** [review]

**Review by richscott — APPROVED** [review]

**Pull request merged**